README.md

```diff
@@ -56,6 +59,10 @@ Then install using:
 sudo python3 setup.py install
 ```
 
+Alternatively, To install pgmpy, Obtain it from PyPI by using pip
```

Comment:
@souravsingh This seems better to me:  `Alternatively you can install pgmpy using pip`